src/BinaryParsers/PEBinary/PortableExecutable/PE.cs

```diff
@@ -609,15 +613,10 @@ internal static ManagedPlatform ComputeIsDotNetCore(MetadataReader metadataReade
                     {
                         return ManagedPlatform.DotNetStandard;
                     }
-
-                    default:
-                    {
-                        break;
-                    }
                 }
             }
 
-            throw new InvalidOperationException("Could not identify managed platform.");
+            return ManagedPlatform.Unknown;

```

Comment:
We now no longer will get early insight that we have an unknown platform. Why is not failing fast here a good idea?

Also, in general, we don't want to remove default handlers in switch statements. If you want this change, you should move it into the default handler. I'm not convinced (yet!) it's a good idea to eliminate the throw. What are your thoughts?

Reply:
This is preventing me to analyze some binaries...

When we have a dll that has aacorlib as reference, we throw an exception, but in that case, the dll is a valid PE to analyze.

we can also discuss later :)

Reply:
send me a binary that throws bad references and I will take a look. the issue here is that someone is now produce PEs where we fundamentally don't understand the platform. we need to fix this! note that it is these binaries in many cases that *are* in fact .net core! and so it's especially important to chase that down given the purpose of this helper.

this is one of those places where the lack of curated metadata is a real disadvantage. i.e., our detections here are heuristic based and fragile over time, as frameworks and compilers change....

Reply:
added aacorlib as netframework.